AppBroker.Generators/AppBroker.Generators.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<LangVersion>9.0</LangVersion>
+
+		<!-- Special properties for analyzer packages. -->
+		<IncludeBuildOutput>false</IncludeBuildOutput>
+		<!--<IncludeSymbols>false</IncludeSymbols>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <UseNETCoreGenerator>true</UseNETCoreGenerator>
+    <IsRoslynComponent>true</IsRoslynComponent>-->
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.10.0" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.10.0" PrivateAssets="all" GeneratePathProperty="true" />
+		<None Include="$(PkgMicrosoft_CodeAnalysis_CSharp)\lib\netstandard2.0\*.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+		<None Include="$(OutputPath)$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+	</ItemGroup>
+
+   <PropertyGroup>
+        <GetTargetPathDependsOn>$(GetTargetPathDependsOn);BlubbiChanged-GetDependencyTargetPaths</GetTargetPathDependsOn>
```

Comment:
BlubbiChanged?

Reply:
Copied from you, doesn't have any effect, so keep it as a credit ;)